Add meeting notes for 15/7/25 (#6453)
Couchers-org/couchers: docs/meetings/dev/2025-07-15.md

# Weekly Dev meeting

When: 15/7/25, 20:00 UTC.
Present: Aapeli, Andrei, Bruna, Chris, Dieu, Jesse, Krishna, Nicole, Yannic
Scribe: Aapeli

# Notes

* Nicole: some background on deadlines and priorities

>   - two milestones: Pumpkin Drop (Oct 1), Winter Drop (Jan 8) -- these are internal names

Reviewer: ```suggestion
  - two milestones: Pumpkin Drop (Oct 1), Snowball Drop (Jan 8) -- these are internal names
```

    * fix default "about me" template on profile
  - Growth
    * translations
    * merch
    * more info pages (what is couch surfing? couchers vs ...)
    * social share links on blog posts
    * team page updates
  - Moderation
    * Move main console screen on-platform
    * Implement on-platform event invite decision flow
    * Unified Moderation System part 1: "moderation state table"
  - ...and a bit more...
* Andrei: recommendation on monitoring our competition and picking up good ideas from there
* Jesse: indeed, many of us very familiar with competition and a lot of our ideas are based on improvements to other platforms
* Bruna: idea on a trello board where trusted community members can suggest ideas
* Discussion on a feature request board -- e.g. logchimp
* Aapeli: data science team
* Aapeli: upcoming board meeting

* What everyone's working on:
  - Chris: onboarding translators, bug hunting
  - Yannic: data science, boosting/ranking changes
  - Krishna: working on "how did you hear about us" on signup; looking at postgres issues, open to helping out on data engineering
  - Andrei: finished some work he was working on; will talk with Aapeli on what else he can work on

>   - Nicole: finished landing page fixes, lots of translation stuff (updated job ad, translation picker updates, status page, added sv+ua); working next on report reasons fix, basic mod page, maintenance

Reviewer: ```suggestion
  - Nicole: finished landing page fixes, made tickets and organized board for new priorities, lots of translation stuff (updated job ad, translation picker updates, status page, added sv+uk); working next on report reasons fix, basic mod page, maintenance, bug fixes from Chris, interviews for new volunteers
```

  - Bruna: have been moving; will look at merch for Thu meeting

>   - Dieu: busy with job search stuff; looking at working on a task by the end of the month, Nicole will help her with finding one

Reviewer: ```suggestion
  - Dieu: busy with job search stuff; looking at working on a task by the end of the month, Nicole will help with finding one
```
